src/styles/utils/variables.scss

```diff
@@ -0,0 +1,2 @@
+$-elements-margin: 140px;
```

Comment:
It would be beneficial to transfer the variables that are solely connected to one block to the file associated with that specific block. Not clear to which element does the variable refer